openwisp_monitoring/device/api/serializers.py

```diff
@@ -20,6 +26,77 @@ class Meta:
         model = DeviceMonitoring
 
 
+class WifiClientSerializer(FilterSerializerByOrgManaged, ValidatedModelSerializer):
+    class Meta:
+        fields = [
+            "mac_address",
```

Comment:
maintain consistency of the quoting style with the rest of the codebase, we use single quotes